validation/src/test/scala/hmda/validation/rules/lar/validity/V525Spec.scala

```diff
@@ -0,0 +1,26 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.{ Loan, LoanApplicationRegister }
+import hmda.validation.dsl.{ Failure, Success }
+import hmda.validation.rules.lar.LarEditCheckSpec
+import org.scalacheck.Gen
+
+class V525Spec extends LarEditCheckSpec with BadValueUtils {
```

Comment:
I get a compilation error on your branch, it's not finding `BadValueUtils` and `intOutsideRange`. Can you check that you have merged this functionality into your PR?